citesphere/src/main/webapp/WEB-INF/views/home.html

```diff
@@ -29,6 +83,38 @@ <h2>Welcome back, <span th:text="${user.firstName}"></span>!</h2>
 				<div class="panel-body">
 					<a th:href="@{'/auth/group/' + ${group.groupId} + '/items'}"><span th:text="${group.name}"/>
 					(<span th:text="${group.numItems}"/>)</a>
+					<a class="delete-link" style="float: right;" href="" th:data-client-id="${group.groupId}"><i class="fas fa-trash-alt" ></i></button>
+					</a>
+					
+					<!--  <form action="#" th:action="@{'/auth/group/' + ${group.groupId} + '/mongo'}" method="GET">
+					<button class="btn btn-primary btn-lg" style="float: right; "><span>Delete Local copy </span> </button>
+					</form>
+					-->
+				</div>
+				<div class="modal fade" id="client-delete-confirmation" tabindex="-1" role="dialog">
+  					<div class="modal-dialog" role="document">
+    					<div class="modal-content">
+      						<div class="modal-header">
+        						<button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        						<h4 class="modal-title">Delete App</h4>
+      						</div>
+      						<div class="modal-body">
+        						<p>Are you sure you want to delete the local copy of the group? </p>
```

Comment:
Let's make this
"Are you sure you want to delete the local copy of the group? This will trigger a resync."